tests/stylable-proxy.spec.ts

```diff
@@ -0,0 +1,25 @@
+import 'mocha';
+import {expect} from 'chai';
+import stylableProxy from './utils/stylable-proxy';
+
+describe('stylable-proxy', () => {
+  before(() => {
+    require.extensions['.css'] = function mockCss(module) {
+      module.exports = stylableProxy;
+    };
+  });
+
+  it('should apply the proxy function and preserve stylable', () => {
+    const css = require('./test-fixtures/simple.st.css');
+    expect(css.a).to.equal('a');
+    expect(css('root', {state: true}, {prop: 1})).to.eql({
```

Comment:
@barak007 
Now I see that I didn't test the `default` behavior:

```
expect(css.default('root', {state: true}, {prop: 1})).to.eql({
  className: 'root',
  "data-namespace-state": true
});
```

It fails - how should it be handled in my proxy?